cache_slug.yaml

```diff
@@ -0,0 +1,3 @@
+# cache slug should not start or end with any slashes.
```

Comment:
Add more documentation. What's a cache slug? What format must its name follow? Is it meant to be updated automatically or does the releaser do it in every release?